knora-ontologies/knora-base.ttl

```diff
@@ -972,9 +972,7 @@
 
             rdfs:subPropertyOf :systemProperty ;
 
-            rdfs:comment "Indicates when a resource was deleted"@en ;
-
-            :subjectClassConstraint :Resource ;
```

Comment:
Did you remove the constraint because this can apply to a resource and/or a value?